playbooks/existing-server.yml

```diff
@@ -0,0 +1,15 @@
+---
+# existing-server.yml is an advanced provisioning option that doesn't use a genesis
+# role to create a new server and instead applies Streisand to an existing
+# remote server.
+#
+# Try and detect the remote server's provider & apply required workarounds
+- include: provider-detect.yml
```

Comment:
I vote for including `python.yml` up higher. We've got a perfectly working hammer, we use the pefectly working hammer elsewhere, just use the hammer. You'll need py eventually anyhow.